static/src/js/app/Dashboard.vue

```diff
@@ -1,28 +1,13 @@
 <template>
   <div class="dashboard">
-    <Modal
-      ref="Modal"
-      title="Add Word to Personal Vocabulary List"
-      description="Modal to quick add vocab"
-      ><QuickAddVocabForm ref="QuickAddVocabForm"
-    /></Modal>
-    <button type="button" class="btn btn-primary" @click="openModal">
-      Quick Add Vocab
-    </button>
+    UNDER CONSTRUCTION
```

Comment:
I suggest we remove this, since we can assume it's under construction.

Reply:
Will do!